app/views/oidc/rubygem_trusted_publishers/index_view.rb

```diff
@@ -18,7 +18,8 @@ def view_template
       end
 
       p do
-        button_to t(".create"), new_rubygem_trusted_publisher_path(rubygem.slug), class: "form__submit", method: :get
+        button_to t(".create") + " Buildkite", new_rubygem_trusted_publisher_path(rubygem.slug), params: {trusted_publisher: "buildkite"}, class: "form__submit", method: :get
```

Comment:
instead of multiple buttons here, what if the trusted publisher new view reloaded the form when the selector for trusted publisher type was changed?

Reply:
I'll have to brush up on how stimulus works, but I'll give it a go!